### PR TITLE
More matrix demo feedback

### PR DESCRIPTION
**Pull request by jessex**
## Description of the change

This addresses another wave of feedback for the revocation analysis matrix from our demo roadshow.

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Towards #133 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Lint rules pass locally
- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/components/charts/new_revocations/RevocationsByRace.js [hunk 1]
```diff
@@ -168,15 +173,15 @@ const RevocationsByRace = (props) => {
   return (
     <div>
       <h4>
-        Revocations by race
+        Revocation rates by race and risk level
```
Comment (jjdressel11) [inline src/components/charts/new_revocations/RevocationsByRace.js]: --> "Revocation rates by race/ethnicity and risk level"

File: src/components/charts/new_revocations/RevocationsByRace.js [hunk 2]
```diff
@@ -168,15 +173,15 @@ const RevocationsByRace = (props) => {
...
         <ExportMenu
           chartId={chartId}
           chart={chart}
-          metricTitle="Revocations by race"
+          metricTitle="Revocation rates by race and risk level"
```
Comment (jjdressel11) [inline src/components/charts/new_revocations/RevocationsByRace.js]: --> "Revocation rates by race/ethnicity and risk level"

File: src/components/charts/new_revocations/RevocationsByViolation.js [hunk 3]
```diff
@@ -69,14 +71,14 @@ const RevocationsByViolation = (props) => {
           xAxes: [{
             scaleLabel: {
               display: true,
-              labelString: 'Violation type',
+              labelString: 'Violation type and condition violated (blue is technical, yellow is law)',
```
Comment (jjdressel11) [inline src/components/charts/new_revocations/RevocationsByViolation.js]: I just pushed back on this on the copy doc, but I think the "(blue is technical, yellow is law)" text is clunky and redundant

File: src/components/charts/new_revocations/RevocationsOverTime.js [hunk 4]
```diff
@@ -118,11 +118,11 @@ const RevocationsOverTime = (props) => {
   return (
     <div>
       <h4>
-        Revocations over time
+        Number of people revoked to prison per month
         <ExportMenu
           chartId={chartId}
           chart={chart}
-          metricTitle="Revocations over time"
+          metricTitle="People revoked to prison by month"
```
Comment (jjdressel11) [inline src/components/charts/new_revocations/RevocationsOverTime.js]: Let's have the export title be consistent with the dashboard title "Number of people revoked to prison per month"

File: src/utils/charts/toggles.js [hunk 5]
```diff
@@ -139,6 +139,34 @@ function standardTooltipForRateMetric(tooltipItem, data) {
   return `${label}: ${getTooltipWithoutTrendline(tooltipItem, data, '%')}`;
 }
 
+function tooltipForRateMetricWithCounts(tooltipItem, data, numerators, denominators) {
```
Comment (jjdressel11) [inline src/utils/charts/toggles.js]: LOVE THIS

File: src/views/tenants/us_mo/Revocations.js [hunk 6]
```diff
@@ -55,7 +55,7 @@ const DEFAULT_METRIC_PERIOD = '12';
 const CHARGE_CATEGORIES = [
   { value: 'All', label: 'All' },
   { value: 'GENERAL', label: 'General' },
-  { value: 'SEX_OFFENDER', label: 'Sex offense' },
+  { value: 'SEX_OFFENDER', label: 'Sex Offense' },
```
Comment (jjdressel11) [inline src/views/tenants/us_mo/Revocations.js]: I think they want this label to be "Sex Offender"

Comment (jjdressel11) [reply src/views/tenants/us_mo/Revocations.js]: Tho I like "Sex Offense" better so maybe just leave it

File: src/components/charts/new_revocations/RevocationsByViolation.js (hunk 3 above)
Comment (jessex) [reply src/components/charts/new_revocations/RevocationsByViolation.js]: I agree, will remove.

File: src/components/charts/new_revocations/RevocationsOverTime.js (hunk 4 above)
Comment (jessex) [reply src/components/charts/new_revocations/RevocationsOverTime.js]: Whoops, missed that update. Will fix!

File: src/views/tenants/us_mo/Revocations.js (hunk 6 above)
Comment (jessex) [reply src/views/tenants/us_mo/Revocations.js]: The issue with "Sex Offender" is that it's a different tense from "Domestic Violence." If we change to "Sex Offender" then consistency would demand we change the latter to something like "Domestic Violence Offender" or something. Given that the title of the dropdown is itself "Supervision Level" and is thus focused on the supervision case itself, I think "Sex Offense" is the right language.

File: src/components/charts/new_revocations/RevocationsByRace.js (hunk 2 above)
Comment (jessex) [reply src/components/charts/new_revocations/RevocationsByRace.js]: Sure thing.

File: src/components/charts/new_revocations/RevocationsByRace.js (hunk 1 above)
Comment (jessex) [reply src/components/charts/new_revocations/RevocationsByRace.js]: Sure thing.

File: src/views/tenants/us_mo/Revocations.js (hunk 6 above)
Comment (jjdressel11) [reply src/views/tenants/us_mo/Revocations.js]: I also think we should avoid using the term "Offender" as much as possible

**Comment by jessex** [comment]
Final approval, please! @jjdressel11

**Pull request merged**